components/apitemplate_io/app/apitemplate_io.app.ts

```diff
@@ -1,13 +1,279 @@
 import { defineApp } from "@pipedream/types";
+import { axios } from "@pipedream/platform";
 
 export default defineApp({
   type: "app",
   app: "apitemplate_io",
-  propDefinitions: {},
+  propDefinitions: {
+    apiEndpoints: {
+      type: "string",
+      label: "API Endpoints",
+      description: "A regional API endpoint is intended for customers in the same region. The data for the request and generated PDFs/images are processed and stored within the region. [see docs](https://apitemplate.io/apiv2/#section/Regional-API-endpoint(s))",
+      optional: true,
+      options: [
+        {
+          label: "Default (Singapore)",
+          value: "https://rest.apitemplate.io",
+        },
+        {
+          label: "Europe (Frankfurt)",
+          value: "https://rest-de.apitemplate.io",
+        },
+        {
+          label: "US East (N. Virginia)",
+          value: "https://rest-us.apitemplate.io",
+        },
+        {
+          label: "Australia (Sydney)",
+          value: "https://rest-au.apitemplate.io",
+        },
+        {
+          label: "Alternative Default (Singapore)",
+          value: "https://rest-alt.apitemplate.io",
+        },
+        {
+          label: "Alternative Europe (Frankfurt)",
+          value: "https://rest-alt-de.apitemplate.io",
+        },
+        {
+          label: "Alternative US East (N. Virginia)",
+          value: "https://rest-alt-us.apitemplate.io",
+        },
+      ],
+    },
+    exportType: {
+      type: "string",
+      label: "Export Type",
+      description: "Either `file` or `json`(Default).",
+      optional: true,
+      options: [
+        "json",
+        "file",
+      ],
```

Comment:
These are not too crowded but optionally we can move these and other constant options to constants file too